pheweb/load/make_gene_aliases_sqlite3.py

```diff
@@ -82,18 +84,28 @@ def download_gene_aliases() -> None:
         db.executemany('INSERT INTO gene_aliases VALUES (?,?)', sorted(get_gene_aliases().items()))
     aliases_tmp_filepath.replace(aliases_filepath)
 
+
 def run(argv:List[str]) -> None:
     if '-h' in argv or '--help' in argv:
         print('Make a database of all gene names and their aliases for easy searching.')
         exit(1)
-
+    
     # This needs genes for filtering
-    if not Path(get_filepath('genes', must_exist=False)).exists():
+    gene_filepath = Path(get_filepath('genes', must_exist=False))
+    if not gene_filepath.exists():
         print('Downloading genes')
-        from . import download_genes
         download_genes.run([])
+    else:
+        print("getting an existing bed file: {}".format(gene_filepath))
+        genes = [{'canonical': canonical, 'ensg':ensg} for _,_,_,canonical,ensg in get_gene_tuples_with_ensg()]
+        if not len({g['ensg'] for g in genes}) == len(genes) or not len({g['canonical'] for g in genes}) == len(genes):
```

Comment:
I'm thinking that actually maybe better to error out. This will for sure be silently missed in case there is an error in the provided file.